test/framework/kubernetes.go

```diff
@@ -158,8 +205,8 @@ func loadResource(namespace, uri string, resourceRef meta.ResourceObject, before
 	return nil
 }
 
-// WaitForAllPodsToContainTextInLog waits for pods of specified deployment config to contain specified text in log
-func WaitForAllPodsToContainTextInLog(namespace, dcName, logText string, timeoutInMin int) error {
+// WaitForAllPodsByDeploymentConfigToContainTextInLog waits for pods of specified deployment config to contain specified text in log
+func WaitForAllPodsByDeploymentConfigToContainTextInLog(namespace, dcName, logText string, timeoutInMin int) error {
```

Comment:
Nitipick, but we don't have DeploymentConfigs any more :p

Reply:
I know :) I left it there just for the case.